### ISSUE TITLE
AltStore being detected as Trojan

### ISSUE THREAD

**Issue by Chocolateismybaby** [rileytestut/AltStore#114]
My antivirus throws a trojan warning when I open the MSI package, so I ignored it and installed but it also threw the same warning when I tried running AltServer.exe

**Comment by marleyqp** [comment]
Same here

**Comment by Terrain2** [comment]
What antivirus is that? Try uploading the file to virustotal, because it’s likely a false positive in just your antivirus

Though i can certainly see why it would be detected, it’s waiting for a command from a remote device, upon which it will download a (potentially malicious, but actually safe) file and do some stuff to it using other programs on the computer

Of course this is leaving out important bits how that remote device is an application the user installed on purpose knowing that it will send commands to a program on their pc, and the fact that all it does is sign an ipa file, which is a perfectly safe operation, but the antivirus wouldn’t know this

TL;DR false positive

**Comment by noah978** [comment]
This is just a false positive. I assure you, there is no malware if you are sure to download from the official site: https://altstore.io

Closing this issue.
